### DISCUSSION TITLE
Add a "Getting better" difficulty/mod that fails/restarts the current play the moment you can't pass your previous best

### DISCUSSION THREAD

**Discussion by chrislampro** [ppy/osu#29471] [Ideas]
A mod that stops or restarts your play if your current score cannot surpass your previous best, even if you FC the rest of the map perfectly. For example, if you're halfway through a play and the game detects that your potential score won't exceed your previous best, it will either fail you or restart the play.

I haven't seen this implemented, nor have I seen it suggested. The only thing that comes close is "Accuracy Challenge" but you have to set the accuracy yourself and you can't set it as decimals

**Comment by Jinkku** [comment]
I guess this is kinda simular of what "Accuracy Challenge" is going for. Not a bad idea, I'm upvoting this ^^

**Reply by chrislampro** [reply]
Thank you!

**Comment by chrislampro** [comment]
Here after the update video. Hopefully this will be part of the next one. I am not familiar with github etiquette so please excuse me if I've done something wrong.

**Comment by chrislampro** [comment]
Good job on the newest video. Hopefully this will be added in the next one.
